tool/microkit/src/loader.rs

```diff
@@ -247,40 +268,61 @@ impl<'a> Loader<'a> {
+            loader_buf
+                .write_all(region_metadata_bytes)
+                .expect("Failed to write region metadata to loader");
         }
 
         // Now we can write out all the region data
         for (_, data) in &self.regions {
-            loader_buf.write_all(data).expect("Failed to write region data to loader");
+            loader_buf
+                .write_all(data)
+                .expect("Failed to write region data to loader");
         }
 
         loader_buf.flush().unwrap();
     }
 
-    fn setup_pagetables(elf: &ElfFile, first_vaddr: u64, first_paddr: u64) -> [(u64, u64, [u8; PAGE_TABLE_SIZE]); 5] {
-        let (boot_lvl1_lower_addr, boot_lvl1_lower_size) = elf.find_symbol("boot_lvl1_lower").expect("Could not find 'boot_lvl1_lower' symbol");
-        let (boot_lvl1_upper_addr, boot_lvl1_upper_size) = elf.find_symbol("boot_lvl1_upper").expect("Could not find 'boot_lvl1_upper' symbol");
-        let (boot_lvl2_upper_addr, boot_lvl2_upper_size) = elf.find_symbol("boot_lvl2_upper").expect("Could not find 'boot_lvl2_upper' symbol");
-        let (boot_lvl0_lower_addr, boot_lvl0_lower_size) = elf.find_symbol("boot_lvl0_lower").expect("Could not find 'boot_lvl0_lower' symbol");
-        let (boot_lvl0_upper_addr, boot_lvl0_upper_size) = elf.find_symbol("boot_lvl0_upper").expect("Could not find 'boot_lvl0_upper' symbol");
+    fn setup_pagetables(
+        elf: &ElfFile,
+        first_vaddr: u64,
+        first_paddr: u64,
+    ) -> [(u64, u64, [u8; PAGE_TABLE_SIZE]); 5] {
+        let (boot_lvl1_lower_addr, boot_lvl1_lower_size) = elf
+            .find_symbol("boot_lvl1_lower")
+            .expect("Could not find 'boot_lvl1_lower' symbol");
+        let (boot_lvl1_upper_addr, boot_lvl1_upper_size) = elf
+            .find_symbol("boot_lvl1_upper")
+            .expect("Could not find 'boot_lvl1_upper' symbol");
+        let (boot_lvl2_upper_addr, boot_lvl2_upper_size) = elf
+            .find_symbol("boot_lvl2_upper")
+            .expect("Could not find 'boot_lvl2_upper' symbol");
+        let (boot_lvl0_lower_addr, boot_lvl0_lower_size) = elf
+            .find_symbol("boot_lvl0_lower")
+            .expect("Could not find 'boot_lvl0_lower' symbol");
+        let (boot_lvl0_upper_addr, boot_lvl0_upper_size) = elf
+            .find_symbol("boot_lvl0_upper")
+            .expect("Could not find 'boot_lvl0_upper' symbol");
 
         let mut boot_lvl0_lower: [u8; PAGE_TABLE_SIZE] = [0; PAGE_TABLE_SIZE];
         boot_lvl0_lower[..8].copy_from_slice(&(boot_lvl1_lower_addr | 3).to_le_bytes());
 
         let mut boot_lvl1_lower: [u8; PAGE_TABLE_SIZE] = [0; PAGE_TABLE_SIZE];
         for i in 0..512 {
             #[allow(clippy::identity_op)] // keep the (0 << 2) for clarity
-            let pt_entry: u64 =
-                ((i as u64) << AARCH64_1GB_BLOCK_BITS) |
+            let pt_entry: u64 = ((i as u64) << AARCH64_1GB_BLOCK_BITS) |
```

Comment:
This one makes it harder to read... If we don't follow rustfmt for some things, will the CI fail?

Reply:
`#[cfg_attr(rustfmt, rustfmt_skip)]` could work